Polling accessory status information

pvanessen opened the pull request:
Hey! I love this plugin. The only thing I have notices it is not doing is to keep the information about the current state of an accessory. Let's say, a switch is turned on. My API returns the {"getOn": "true"} to the plugin and HomeKit shows the switch turning on. But when the HomeKit app is updating, it goes back to "Off" although my API logic still has it "on". 
What am I missing?

tasict committed:
Add option characteristic for service

tasict committed:
Rename project

tasict committed:
Add .DS_Store to gitignore

tasict committed:
Fix crash bug

tasict committed:
Fix crash bug

Commit:
Add homekit extension

Commit:
correct code

Commit:
Add HomeKit Extension

tasict committed:
Refine code

tasict committed:
Refine code

tasict committed:
Refine code

tasict committed:
Refine code

tasict committed:
Fix Bugs

tasict committed:
Fix Bugs

Commit:
Refine code

Commit:
Add service extension.

tasict committed:
Refine code

budjoe commented:
Can you add a note to this project that indicates that the response to the Get should be a JSON file.
For devices with a single value it should look like {"value": xxx}.
No name for the value, just value.
I haven't added any devices with multiple values to submit, but I would assume it would be {"value1": xxx, "value2": xxx} and so on.

JimJamUrCode committed:
Create API Behavior instruction (#1)

It would be great if you could create some basic instructions on how to setup the server that is actually controlling the device.

It looks like you piece together a URL in code so I imagine there is some strict standard that the server must follow.

tasict committed:
Fix get name error

tasict committed:
Merge branch 'master' of github.com:tasict/homebridge-http-accessory
